File: 10871/10871.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    10871.py                                           :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2021/09/27 14:50:26 by jihoolee          #+#    #+#              #
#    Updated: 2021/09/27 14:50:31 by jihoolee         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

_, max_num = map(int, input().split())
nums = list(map(int, input().split()))
for num in nums:
	if (num < max_num) : print(num, end=' ')

File: 10871/10871.cpp
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   10871.cpp                                          :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2021/09/27 14:50:16 by jihoolee          #+#    #+#             */
/*   Updated: 2021/09/27 14:50:20 by jihoolee         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

#include <iostream>
using namespace std;

int main(void)
{
	int		len;
	int		max;
	int		num;

	cin >> len >> max;
	for (int i = 0; i < len; i++)
	{
		cin >> num;
		if (num < max)
			cout << num << ' ';
	}
	return (0);
}
